x/evm/types/params.go

```diff
@@ -153,10 +146,23 @@ func validateEIPs(i interface{}) error {
 		return fmt.Errorf("invalid EIP slice type: %T", i)
 	}
 
+	uniqueEIPs := make(map[int64]struct{})
```

Comment:
See the validatePrecompiles function below for reference

Reply:
I precisely used this implementation because it saves memory -> check this StackOverflow answer for reference: https://stackoverflow.com/a/9251352

Reply:
@MalteHerrmann we only need one map